### PR TITLE
Add nodejs send wrapper methods

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Add send wrapper methods so it's easier for users

## Links to any relevant issues

Does the nodejs binding part for https://github.com/iotaledger/iota-sdk/issues/544, Python will be done in another PR

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Running an example

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: bindings/nodejs/lib/wallet/account.ts [hunk 1]
```diff
@@ -116,6 +116,20 @@ export class Account {
         return Output.parse(JSON.parse(response).payload) as NftOutput;
     }
 
+    /**
+     * A generic `burn()` function that can be used to prepare to burn native tokens, nfts, foundries and aliases.
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/wallet/account.ts]: Doesn't just "prepare"?

File: bindings/nodejs/lib/wallet/account.ts [hunk 2]
```diff
@@ -116,6 +116,20 @@ export class Account {
...
+     * @param burn The outputs to burn
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/wallet/account.ts]: Not necessarily outputs...

File: bindings/nodejs/lib/wallet/account.ts [hunk 3]
```diff
@@ -116,6 +116,20 @@ export class Account {
...
+     * @param transactionOptions The options to define a `RemainderValueStrategy`
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/wallet/account.ts]: I know it was probably already there but I wouldn't mention specific options, sounds unmaintainable.

File: bindings/nodejs/lib/wallet/account.ts [hunk 4]
```diff
@@ -233,6 +248,20 @@ export class Account {
         return plainToInstance(Transaction, parsed.payload);
     }
 
+    /**
+     * Consolidate basic outputs with only an `AddressUnlockCondition` from an account
+     * by sending them to an own address again if the output amount is greater or
+     * equal to the output consolidation threshold.
+     * @param force Force consolidation on addresses where the threshold isn't met.
+     * @param outputConsolidationThreshold A default threshold is used if this is omitted.
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/wallet/account.ts]: These params don't exist anymore

File: bindings/nodejs/lib/wallet/account.ts [hunk 5]
```diff
@@ -262,6 +291,22 @@ export class Account {
         );
     }
 
+    /**
+     * `createAliasOutput` creates an alias output
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/wallet/account.ts]:
```suggestion
     * Creates an alias output.
```

File: bindings/nodejs/lib/wallet/account.ts [hunk 6]
```diff
@@ -825,6 +916,23 @@ export class Account {
         );
     }
 
+    /**
+     * Create a native token.
+     *
+     * @param params The options for creating a native token.
+     * @param transactionOptions The options to define a `RemainderValueStrategy`
+     * or custom inputs.
+     * @returns The creating transaction and the token ID.
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/wallet/account.ts]: But it returns a `Transaction`?

File: bindings/nodejs/lib/wallet/account.ts [hunk 7]
```diff
@@ -970,6 +1093,21 @@ export class Account {
...
 
+    /**
+     * Send a transaction.
+     *
+     * @param outputs Outputs to use in the transaction.
+     * @param options The options to define a `RemainderValueStrategy`
+     * or custom inputs.
+     * @returns The transaction data.
+     */
+    async transaction(
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/wallet/account.ts]: Feels a bit weird as a method name

Comment (Thoralf-M) [reply bindings/nodejs/lib/wallet/account.ts]: Should I add `send` prefix?

Comment (thibault-martinez) [reply bindings/nodejs/lib/wallet/account.ts]: Yeah I think that would be better

File: bindings/nodejs/CHANGELOG.md [hunk 8]
```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.1.0 - 2023-MM-DD
+
+### Added
+
+- `Account::{burn(), consolidateOutputs(), createAliasOutput(), meltNativeToken(), mintNativeToken(), createNativeToken(), mintNfts(), transaction(), sendNativeTokens(), sendNft()}` methods;
```
Comment (Thoralf-M) [inline bindings/nodejs/CHANGELOG.md]:
```suggestion
- `Account::{burn(), consolidateOutputs(), createAliasOutput(), meltNativeToken(), mintNativeToken(), createNativeToken(), mintNfts(), sendTransaction(), sendNativeTokens(), sendNft()}` methods;
```

File: bindings/nodejs/lib/wallet/account.ts [hunk 9]
```diff
@@ -116,10 +116,24 @@ export class Account {
         return Output.parse(JSON.parse(response).payload) as NftOutput;
     }
 
+    /**
+     * A generic `burn()` function that can be used to burn native tokens, nfts, foundries and aliases.
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/wallet/account.ts]:
```suggestion
     * A generic function that can be used to burn native tokens, nfts, foundries and aliases.
```

File: bindings/nodejs/lib/wallet/account.ts [hunk 10]
```diff
@@ -237,8 +252,20 @@ export class Account {
...
+        params: ConsolidationParams,
+    ): Promise<Transaction> {
+        return (await this.prepareConsolidateOutputs(params)).send();
+    }
+
+    /**
+     * Consolidate basic outputs with only an `AddressUnlockCondition` from an account
+     * by sending them to an own address again if the output amount is greater or
+     * equal to the output consolidation threshold.
+     * @param params FConsolidation options.
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/wallet/account.ts]:
```suggestion
     * @param params Consolidation options.
```

**Review by thibault-martinez — APPROVED** [review]

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
Failing CI

**Review by thibault-martinez — APPROVED** [review]

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
Conflicts

**Review by abdulmth — DISMISSED** [review]

**Review by abdulmth — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**